docs/tutorials/flight_controller_telemetry_tutorial.rst

```diff
@@ -0,0 +1,395 @@
+in the code block below.
+
+.. code::
+
+    motor_test test -m 1 -p 5
+    INFO  [motor_test] motor 1 set to 0.05
+    nsh> 
+
+The image below shows an example of what the instrument panel may look like when the module is spinning. 
+
+.. figure:: ../_static/tutorial_images/fc_telemetry_tutorial/px4_dshot_instrument_panel_spinning.png
+    :align: center
+    :width: 40%
+    :alt: QGroundControl Instrument Panel Showing DSHOT Telemetry When the Module is Spinning
+
+    QGroundControl Instrument Panel Showing DSHOT Telemetry When the Module is Spinning
+
+Note that if you are using the default settings for MOT_POLE_COUNT, the RPM shown on the instrument panel will be significantly higher than the actual RPM of the module. This is because PX4 expects the module
+to report ERPM, but Vertiq modules report RPM directly as covered in the :ref:`ESC Telemetry <manual_telemetry>` section of the Feature Reference Manual.
+To learn how to adjust the MOT_POLE_COUNT to properly show RPM, see the :ref:`fc_telemetry_erpm_to_rpm` section below.
+
+.. _fc_telemetry_erpm_to_rpm:
+
+Converting ERPM to RPM Workaround
+##################################
+As of v1.13.3, PX4 flight controllers expect the DSHOT telemetry messages to match the `KISS ESC standard <https://www.rcgroups.com/forums/showatt.php?attachmentid=8524039&d=1450424877>`_.
+That means that it expects the module to send ERPM/100, while Vertiq modules directly send RPM as covered in :ref:`ESC Telemetry <manual_telemetry>`.
+
+To convert from ERPM to RPM, the ERPM value should be divided by the pole count of the motor divided by 2. The flight controller is expecting to receive 
+ERPM/100, so when it receives telemetry it tries to calculate RPM using the following formula shown below. This calculation can also be
+seen in the `PX4 code for DSHOT parsing <https://github.com/PX4/PX4-Autopilot/blob/be03b7f098800d51281142cb0319cc37d36cfed2/src/drivers/dshot/DShot.cpp#L228>`_.
+
+.. figure:: ../_static/tutorial_images/fc_telemetry_tutorial/px4_erpm_to_rpm_formula.png
+    :align: center
+    :width: 40%
+    :alt: ERPM to RPM Conversion
+
+    ERPM to RPM Conversion
+
+Because Vertiq modules send RPM instead of ERPM, this conversion is unnecessary, and actually leads to the flight controller calculating the incorrect RPM by default.
+However, there is a workaround available on PX4 that makes it possible for the flight controller to calculate the correct RPM when using Vertiq modules. 
+
+The conversion includes the number of poles on the motor. By setting the number of poles to 200, the conversion simplifies down to multiplying the reported RPM by 1.
+This means that the received RPM is used without any conversion, which is what we want when using Vertiq modules. Because of this, it is recommended to set the MOT_POLE_COUNT 
+to 200, as shown below, when using DSHOT telemetry with a Vertiq module. This is not the actual number of poles in the module, but the actual number of poles is irrelevant

```

Comment:
I feel like a 200 pole motor would be cool to look at...

Reply:
Indeed